ExoPlayerAdapter/src/main/java/com/mux/stats/sdk/muxstats/internal/BandwidthMetric.kt

```diff
@@ -42,14 +39,14 @@ internal open class BandwidthMetric(val player: ExoPlayer, val collector: MuxSta
      * @return segment that failed to load.
      */
     open fun onLoadError(loadTaskId: Long, e: IOException): BandwidthMetricData {
-        var segmentData: BandwidthMetricData? = loadedSegments.get(loadTaskId)
+        var segmentData: BandwidthMetricData? = loadedSegments[loadTaskId]
         if (segmentData == null) {
             segmentData = BandwidthMetricData()
             // TODO We should see how to put minimal stats here !!!
         }
-        segmentData.setRequestError(e.toString())
+        segmentData.requestError = e.toString()
         // TODO see what error codes are
-        segmentData.requestErrorCode == -1
```

Comment:
!

Reply:
I think it should be an assignment. There's no implicit behavior here, it's just evaluating and discarding the result